src/app/dataset/depublication/sortable-header/index.ts

```diff
@@ -0,0 +1 @@
+export * from './sortable-header.component';
```

Comment:
Do you foresee this ever being reused? Perhaps then sortable-group and sortable-header should be moved out of the depublication directory and somewhere shared? Like you're doing for the file upload component.

Reply:
I mention my thinking on this above: moving it to "shared" would imply that it's shared when it isn't really.  That said... the file upload component is treated separately...

Reply:
As I commented above, let's leave it like it is for now.